utils/coverage/coverage-touch-tests

```diff
@@ -86,7 +86,7 @@ def main():
     console.setLevel(level=args.log.upper())
     logging.debug(args)
     environment = os.environ.copy()
-    environment['GIT_EXTERNAL_DIFF'] = script_path
+    environment['GIT_EXTERNAL_DIFF'] = script_path.replace('\\', '/')
```

Comment:
I could've sworn that that there was a path utility to convert the slashes in python.

Reply:
It looks like [`os.path.normpath`](
https://docs.python.org/2/library/os.path.html#os.path.normpath) does what you're looking for.

Reply:
It does the opposite if I read the documentation right: “On Windows, it converts forward slashes to backward slashes.”. In this case, for some reason, `GIT_EXTERNAL_DIFF` was removing the backward slashes from the path, but worked when I provided the forward slashes. I can add a comment explaining why it is needed.